### DISCUSSION TITLE
I can't find the zksync plugin on remix

### DISCUSSION THREAD

**Discussion by cyber-jev** [Cyfrin/foundry-full-course-cu#2066] [Q&A] [answered]
For some reasons I can't find the zksync plugin on remix in other to deploy my SimpleStorage.sol contract. I can't seem to find it searching the plugins section in remix.

I don't know if this is an error or something, I would really appreciate some help here. 
Thanks in advance

**Comment by EngrPips** [comment, marked as answer]
Hello @cyber-jev, Please try another browser and see if the problem persists. It seems weird that you didn't see the plugin.

**Reply by cyber-jev** [reply]
thank you so much EngrPips, I found the issue. 
so I switched my internet and then I was able to see zksync in the plugins in remix.

It is a weird one, but the issue was from my internet service provider. I switched to my wifi and boom I can see the zksync plugin that I could not see earlier with the other internet.

Issue Resolved

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out. Keep crushing it.

**Reply by ninecatlife** [reply]
> thank you so much EngrPips, I found the issue. so I switched my internet and then I was able to see zksync in the plugins in remix.
> 
> It is a weird one, but the issue was from my internet service provider. I switched to my wifi and boom I can see the zksync plugin that I could not see earlier with the other internet.
> 
> Issue Resolved

had the same issue. didn't know what to do until i saw this thread. In my own case, i had to put on my VPN to see the zkysnc plugin on remix

**Reply by EngrPips** [reply]
> > thank you so much EngrPips, I found the issue. so I switched my internet and then I was able to see zksync in the plugins in remix.
> > It is a weird one, but the issue was from my internet service provider. I switched to my wifi and boom I can see the zksync plugin that I could not see earlier with the other internet.
> > Issue Resolved
> 
> had the same issue. didn't know what to do until i saw this thread. In my own case, i had to put on my VPN to see the zkysnc plugin on remix

I am glad you figured it out. Keep crushing it.
